release notes/v0.4.0.md

```diff
@@ -0,0 +1,117 @@
+
+- Add `locator.fill` will fill an input with the supplied text ([#380](https://github.com/grafana/xk6-browser/pull/380), [#383](https://github.com/grafana/xk6-browser/pull/383)).
+
+- Add `locator.focus` will focus on an uniquely identifiable element ([#388](https://github.com/grafana/xk6-browser/pull/388)).
+
+- Add `locator.getAttribute` will return the named attribute of a selected locator ([#390](https://github.com/grafana/xk6-browser/pull/390)).
+
+- Add `locator.innerHTML` which will return the html of the given selector ([#392](https://github.com/grafana/xk6-browser/pull/392)).
+
+- Add `locator.innerText` will return the `innerText` from the locator ([#393](https://github.com/grafana/xk6-browser/pull/393)).
+
+- Add `locator.textContent` will return the text from the locator ([#394](https://github.com/grafana/xk6-browser/pull/394)).
+
+- Add `locator.inputValue` will return the text inputted in an input field ([#395](https://github.com/grafana/xk6-browser/pull/395)).
+
+- Add `locator.selectOption` will select the given option in a select element ([](https://github.com/grafana/xk6-browser/pull/397)).
+
+- Add `locator.press` which will "push" the specified keyboard key ([#398](https://github.com/grafana/xk6-browser/pull/398)).
+
+- Add `locator.type` which will "type" the text into the input element ([#399](https://github.com/grafana/xk6-browser/pull/399)).
+
+- Add `locator.hover` which will bring into view the selected element ([#400](https://github.com/grafana/xk6-browser/pull/400)).
+
+- Add `locator.tap` which will bring the into view the selected element and tap on it ([#401](https://github.com/grafana/xk6-browser/pull/401)).
+
+- Add `locator.dispatchEvent` which will dispatch the specified event to the selected element ([#405](https://github.com/grafana/xk6-browser/pull/405)).
+
+- Add `locator.waitFor` which will wait for the element selected to meet the specified state ([#406](https://github.com/grafana/xk6-browser/pull/406)).
+
+
+## Bugs fixed
+
+- Fix zombie browser processes by closing browser processes when the context is canceled ([#306](https://github.com/grafana/xk6-browser/pull/306)).
+
+- Fix so that the browser's temporary directory is cleaned up once the test completes ([#323](https://github.com/grafana/xk6-browser/pull/323), [#409](https://github.com/grafana/xk6-browser/pull/409)).
+
+  There is a small chance that the temporary directory will not be cleaned and a ticket has been opened to track the next step/fix ([#403](https://github.com/grafana/xk6-browser/issues/403)).
+
+- Hold modifier key (e.g. `Shift`) now works when working with `keyboard.down` and `keyboard.up` ([#325](https://github.com/grafana/xk6-browser/pull/325)).
+
+- Fixed `page.content()` so that it no longer panics ([#327](https://github.com/grafana/xk6-browser/pull/327)).
+
+- Fix timeout detection (callApiWithTimeout) ([#411](https://github.com/grafana/xk6-browser/pull/411)).
+
+
+## Improvements
```

Comment:
Slow-motion removals are missing (behavioral change). For example https://github.com/grafana/xk6-browser/commit/491d5da0112060b4c9a8b488df12c85acc5c2bdf. Those are tracked in #234.